compiler/rustc_attr/src/builtin.rs

```diff
@@ -1,10 +1,14 @@
 //! Parsing and validation of builtin attributes
 
-use rustc_ast::{self as ast, Attribute, Lit, LitKind, MetaItem, MetaItemKind, NestedMetaItem};
+use rustc_ast::{
+    self as ast, node_id::CRATE_NODE_ID, Attribute, Lit, LitKind, MetaItem, MetaItemKind,
+    NestedMetaItem,
+};
```

Comment:
Nit: could you avoid multi-line imports and use the same style as elsewhere in this file?